client/src/selectors/issues.js

```diff
@@ -0,0 +1,5 @@
+export const getIssue = state => (
```

Comment:
I think we can simplify this to

`export const getIssue = state => state.issues.length && state.issues[state.issues.length - 1];`